### PR TITLE
Bump the docker image version to 21

### PR DESCRIPTION
**Pull request by petroav**

### CODE REVIEW

File: base-images-tag.json [hunk 1]
```diff
@@ -1,3 +1,3 @@
 {
-	"base_images_tag": "7"
+	"base_images_tag": "21"
```
Comment (rschlussel) [inline base-images-tag.json]: wow, these hadn't been changed in a while!

**Comment by ebd2** [comment]
Please add detail to the commit message explaining what changed and why we're updating.

Assuming Travis passes, LGTM otherwise.

File: base-images-tag.json (hunk 1 above)
Comment (ebd2) [inline base-images-tag.json]: The new images are 3x more terrific than the old ones ;-)

**Comment by petroav** [comment]
Travis passed. Merged!

**Pull request closed without merging**